atom/browser/resources/win/dpi_aware.manifest

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<assembly xmlns="urn:schemas-microsoft-com:asm.v1" manifestVersion="1.0">
+  <asmv3:application xmlns:asmv3="urn:schemas-microsoft-com:asm.v3">
+    <asmv3:windowsSettings xmlns="http://schemas.microsoft.com/SMI/2005/WindowsSettings">
+      <!-- See: https://docs.microsoft.com/en-us/windows/desktop/sbscs/application-manifests#dpiAware -->
+      <dpiAware>true/pm</dpiAware>
+      <disableWindowFiltering xmlns="http://schemas.microsoft.com/SMI/2011/WindowsSettings">true</disableWindowFiltering>
```

Comment:
As per description, `disableWindowFiltering ` is not meant to be in this manifest ?

Reply:
good catch, fixed.